### ISSUE TITLE
Portal fails to restart after with error: object has no attribute 'tools'

### ISSUE THREAD

**Issue by yveskerwyn** [Jumpscale/portal9#99]
#### Installation information

- jumpscale version: 9.3.0
- operating system: Ubuntu 9.3.0
- VM: https://se-gen-1.demo.greenitglobe.com/CBGrid/Virtual%20Machine?id=1842
- SSH: `ssh root@195.143.34.157 -p2200`

#### Detailed description

After configuring ItsYou.online integration using the below statement the portal failed to restart:
```python
redirect_url = "http://{}:{}".format(public_ip_address, 8200)
portal_host.prefab.web.portal.configure(mongodbip='127.0.0.1', mongoport=27017, production=True, client_id='ays-server-clients-org', client_secret=ays_clients_org_secret, scope_organization='ays-portal-users', redirect_address=redirect_url)

portal_host.prefab.web.portal.stop()
portal_host.prefab.web.portal.start()
````

**Error in tmux**:
```
**START**
Traceback (most recent call last):
  File "portal_start.py", line 41, in <module>
    cli(obj={})
  File "/usr/local/lib/python3.5/dist-packages/click/decorators.py", line 17, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "portal_start.py", line 21, in cli
    start()
  File "/usr/local/lib/python3.5/dist-packages/click/decorators.py", line 17, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "portal_start.py", line 35, in start
    server = j.portal.tools.server.get()
AttributeError: 'Portal' object has no attribute 'tools'
'Portal' object has no attribute 'tools'
**ERROR**
```

#### Steps to reproduce

The VM, including the portal was setup using the following script:
```python
import os
app_id = os.environ["APP_ID"]
secret = os.environ["SECRET"]
key_path = os.path.expanduser("~/.ssh/mykey")
ovc_url = "se-gen-1.demo.greenitglobe.com"
location = "se-gen-1"
account_name = "Account_of_Yves"
cloud_space_name = "ays-space2"
ays_host_name = "ays-server"
portal_host_name = "portal"
pubkey_label = "default"
testvdc_name = "myvdc"

ovc_client = j.clients.openvcloud.get(app_id, secret, ovc_url)
account = ovc_client.account_get(account_name, create=False) 

j.clients.ssh.ssh_keys_list_from_agent()
j.clients.ssh.load_ssh_key(path=key_path, create_keys=True)

vdc = account.space_get(cloud_space_name, location)
ays_host = vdc.machine_create(ays_host_name, "mykey", sizeId=3)

internal_ip_address = ays_host.model["nics"][0]["ipAddress"]
public_ip_address = vdc.model["externalnetworkip"]

iyo_user = j.clients.itsyouonline.get_user(app_id, secret)
pubkey = iyo_user.public_keys.get(pubkey_label)
ays_host.prefab.system.ssh.authorize("cloudscalers", pubkey.model["publickey"])
ays_host.prefab.system.ssh.authorize("root", pubkey.model["publickey"])

ays_host.prefab.js9.js9core.install(branch='master')

ays_host.prefab.js9.atyourservice.install(branch='master')

ays_host.prefab.js9.atyourservice.start(host="0.0.0.0")

ays_host.portforward_create(5000, 5000)
public_ays_url = "http://{}:{}".format(public_ip_address, "5000")

ays_host.prefab.js9.atyourservice.configure_api_console(url=public_ays_url)

portal_host = ays_host

portal_host.prefab.web.portal.install(False)

portal_host.prefab.web.portal.start()

portal_host.portforward_create(8200, 8200)

portal_host.prefab.js9.atyourservice.load_ays_space()

portal_host.prefab.web.portal.stop()
portal_host.prefab.web.portal.start()

internal_ays_url = "http://{}:5000".format(internal_ip_address)
api_console_url = "http://{}:5000".format(public_ip_address)
portal_host.prefab.js9.atyourservice.configure_portal(ays_url=internal_ays_url, ays_console_url=api_console_url, portal_name="main")

ays_clients_org_name = "ays-server-clients-org"
api_key_label = "ays-server-api-key"

ays_host.prefab.js9.atyourservice.configure(production=True, organization=ays_clients_org_name)

ays_host.prefab.js9.atyourservice.stop()
ays_host.prefab.js9.atyourservice.start(host="0.0.0.0")

ays_clients_org = iyo_user.organizations.get(ays_clients_org_name)
ays_clients_org_api_key = ays_clients_org.api_keys.get(api_key_label)
ays_clients_org_secret = ays_clients_org_api_key.model["secret"]

redirect_url = "http://{}:{}".format(public_ip_address, 8200)
portal_host.prefab.web.portal.configure(mongodbip='127.0.0.1', mongoport=27017, production=True, client_id='ays-server-clients-org', client_secret=ays_clients_org_secret, scope_organization='ays-portal-users', redirect_address=redirect_url)

portal_host.prefab.web.portal.stop()
portal_host.prefab.web.portal.start()
```

#### Relevant stacktraces

```bash
**START**
Traceback (most recent call last):
  File "portal_start.py", line 41, in <module>
    cli(obj={})
  File "/usr/local/lib/python3.5/dist-packages/click/decorators.py", line 17, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "portal_start.py", line 21, in cli
    start()
  File "/usr/local/lib/python3.5/dist-packages/click/decorators.py", line 17, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "portal_start.py", line 35, in start
    server = j.portal.tools.server.get()
AttributeError: 'Portal' object has no attribute 'tools'
'Portal' object has no attribute 'tools'
**ERROR**
```

**Comment by yveskerwyn** [comment]
solved and tested with 9.2.1.

**Comment by yveskerwyn** [comment]
solved and tested/verfied in 9.2.1
